salt/output/pony.py

```diff
@@ -41,15 +43,20 @@
 
 # Import Python libs
 from __future__ import absolute_import
-import os
 import subprocess
 
 # Import Salt libs
+import salt.utils
 import salt.utils.locales
 
 
+__virtual_name__ = 'pony'
```

Comment:
well, it would have worked since he returns the virtualname from the virtual function. so this would have been a ticking time bomb!

(honestly we would never see an issue here since the virtualname is used to remap the docs from the file name, but the file name is also pony, so this would not have surfaced in any way)